addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -23,7 +23,23 @@ let sequence = 1;
 const getSequence = () => sequence++;
 
 /**
- * @typedef {'audio' | 'camera' | 'screen' } streamType
+ * @typedef {'camera' | 'screen' } VideoType
+ */
+
+/**
+ * @typedef {'audio' | VideoType } StreamType
+ */
+
+/**
+ * @typedef {object} CrossTabActions
+ * @property {boolean} [is_muted]
+ * @property {boolean} [is_deaf]
+ * @property {boolean} [raisingHand]
+ * @property {boolean} [pip]
+ */
+
+/**
+ * @typedef {Object<import("models").id, (import("@mail/discuss/call/common/rtc_session_model").ServerSessionInfo|import("@mail/discuss/call/common/rtc_session_model").SessionInfo)>} sessionInfoMap
```

Comment:
That's a lot of visual LOCs wasted for typedef

We could have single block of comments for all typedefs:

```js
/**
 * @typedef {'camera' | 'screen' } VideoType
 * @typedef {'audio' | VideoType } StreamType
 *
 * @typedef {object} DownloadStates
 * @property {boolean} [audio]
 * @property {boolean} [camera]
 * @property {boolean} [screen]
 *
 * @typedef {object} CrossTabActions
 * @property {boolean} [is_muted]
 * @property {boolean} [is_deaf]
 * @property {boolean} [raisingHand]
 * @property {boolean} [pip]
 */
```